sql/5_0_2-to-6_0_0_upgrade.sql

```diff
@@ -2127,5 +2127,36 @@ ALTER TABLE `form_bronchitis` MODIFY `diagnosis3_bronchitis_form` text;
 ALTER TABLE `form_bronchitis` MODIFY `diagnosis4_bronchitis_form` text;
 #EndIf
 
+#IfColumn patient_data care_team
```

Comment:
before dropping the column, should do a query to copy data over from care_team to care_team_provider